lib/Rename/Adapter/ReferenceFinder/AbstractReferenceRenamer.php

```diff
@@ -78,14 +78,15 @@ protected function renameEdit(Location $location, ?ByteOffsetRange $range, strin
     {
         $referenceDocument = $this->locator->get($location->uri());
 
-        $range = $this->getRenameRange($referenceDocument, $location->offset());
+        // We could probably use the complete range here instead of only using the location
+        $range = $this->getRenameRange($referenceDocument, $location->range()->start());
```

Comment:
not sure what this comment means, let's remove it.

Reply:
The question here was, we already get a range passed into the function. Why is that not the same as the correct rename range?

Reply:
because the range passed in is the user input not the range of the text that should be replaced

Reply:
Ah yeah, will be removed then.